test/python/test_fusedMatMul.py

```diff
@@ -0,0 +1,64 @@
+# ==============================================================================
+#  Copyright 2018 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow bridge fusedMatMul tests.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pytest
+
+import tensorflow as tf
+from tensorflow.python.framework import constant_op
+from tensorflow.python.ops import nn_ops
+from tensorflow.python.ops import nn_impl
+from tensorflow.python.ops import array_ops
+from common import NgraphTest
+from tensorflow.python.framework import dtypes
+
+import numpy as np
+
+
+class TestFusedMatMul(NgraphTest):
+    @pytest.mark.parametrize(
+        ("filename",),
+        (
+            ('fusedmatmul_0.pbtxt',),  # Relu
+            ('fusedmatmul_1.pbtxt',),  # Relu6
+            ('fusedmatmul_2.pbtxt',),  # No activation
+        ))
```

Comment:
gtest: `_FusedMatMul` is an internal op. Not sure if its Cpp exists.
Normal python test. writing a test with `tf.matmul` replaces it with `BatchMatMul` in cpp world, which does not fuse to `_FusedMatMul`.
Python test with pbtxt: Hence using pbtxts to read graphs with `_FusedMatMul` present in them

Reply:
I thought when it was 2x2 matrix, it doesn't go to BatchMatMul, but comes to FusedMatMul. Did you check?

Reply:
I checked, the logs show :
```
OP_placement:   Host    _SOURCE (NoOp)
OP_placement:   Host    _SINK (NoOp)
OP_placement:   Host    Placeholder_5 (Placeholder)
OP_placement:   Host    Placeholder_4 (Placeholder)
OP_placement:   Host    Placeholder_3 (Placeholder)
OP_placement:   Host    BiasAdd_1 (IdentityN)
OP_placement:   nGraph[0]       MatMul_1 (BatchMatMul)
OP_placement:   nGraph[0]       BiasAdd_1_ngraph/_0 (BiasAdd)
```

even for `2x2` it seems to generate `BatchMatMul`. Hence removed the python based graph construction and inserted a `pbtxt` based test